documentation/modules/managing/proc-drain-cleaner-deploying-helm-chart.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// deploying/assembly_deploy-cluster-operator.adoc
+
+[id='deploying-cluster-operator-helm-chart-{context}']
+= Deploying the Strimzi Drain Cleaner using Helm
+
+[role="_abstract"]
+link:https://helm.sh/[Helm] charts are used to package, configure, and deploy Kubernetes resources.
+Strimzi provides a Helm chart to deploy the Strimzi Drain Cleaner.
+
+The Drain Cleaner is deployed on the Kubernetes cluster with the default chart configuration, which assumes that `cert-manager` issues the TLS certificates required by the Drain Cleaner.
+
+You can install the Drain Cleaner with `cert-manager` support or provide your own TLS certificates.
+
+.Prerequisites
+
+* You have xref:drain-cleaner-prereqs-str[downloaded the Strimzi Drain Cleaner deployment files].
+* The Helm client must be installed on a local machine.
+* Helm must be installed to the Kubernetes cluster.
+
+.Default configuration values
+Default configuration values are passed into the chart using parameters defined in a `values.yaml` file.  
+If you don't want to use the default configuration, you can override the defaults when you install the chart using the `--set` argument.
+You specify values in the format `--set key=value[,key=value]`.
+The `values.yaml` file supplied with the Helm deployment files describes the available configuration parameters, including those shown in the following table. 
+
+As you can see, you can override the default image settings.
+You can also set `secret.create` as `true` and add your own TLS certificates instead of using `cert-manager` to generate the certificates. 
+For information on using OpenSSL to generate certificates, see xref:proc-drain-cleaner-certs-{context}[].
+
+Any certificates you add must be renewed before they expire. 
+You can use the configuration to control how certificates are watched for updates using environment variables.
+For more information on how the environment variables work, see xref:proc-drain-cleaner-certs-renewing-{context}[].
+
+.Chart configuration options
+[cols="5m,4,2m",options="header"]
+|===
+
+| Parameter             
+| Description                                                                               
+| Default
+
+| replicaCount
```

Comment:
states `replicas` in the readme

Reply:
Good catch. `replicaCount` is correct. I opened https://github.com/strimzi/drain-cleaner/pull/68 to fix this in the Drain cleaner repo.